src/amo/pages/Guides/index.js

```diff
@@ -173,10 +190,24 @@ export class GuidesBase extends React.Component<InternalProps> {
   }
 }
 
-export const mapStateToProps = (state: AppState) => {
+export const mapStateToProps = (state: AppState, ownProps: Props) => {
```

Comment:
It's very important to keep these mapped props in sync with `InternalProps` so you should add this type hint and address the errors:

```diff
diff --git a/src/amo/pages/Guides/index.js b/src/amo/pages/Guides/index.js
index 555ea482f..354402662 100644
--- a/src/amo/pages/Guides/index.js
+++ b/src/amo/pages/Guides/index.js
@@ -190,7 +190,7 @@ export class GuidesBase extends React.Component<InternalProps> {
   }
 }
 
-export const mapStateToProps = (state: AppState, ownProps: Props) => {
+export const mapStateToProps = (state: AppState, ownProps: Props): $Shape<InternalProps> => {
   const { clientApp, lang } = state.api;
   const { match } = ownProps;
   const { slug } = match.params;

```

Reply:
sorry! I know this is a repeat comment. I'll address this..